db/migrate/sequent/permanently_delete_commands_without_events_v01.sql

```diff
@@ -0,0 +1,12 @@
+CREATE OR REPLACE PROCEDURE permanently_delete_commands_without_events(_aggregate_id uuid, _organization_id uuid)
+LANGUAGE plpgsql AS $$
+BEGIN
+  IF _aggregate_id IS NULL AND _organization_id IS NULL THEN
```

Comment:
I don't think `_organization_id` is used here

Reply:
This parameter is here so it can have a compatible function signature with our internal projects.